src/nextcloud/bin/nextcloud-cron

```diff
@@ -11,6 +11,10 @@ done
 printf "done\n"
 
 while true; do
-	run-php "$SNAP/htdocs/cron.php"
-	sleep 15m
+
+	if [ "$(cronjob_interval)" != "-1" ]; then
+	  run-php "$SNAP/htdocs/cron.php"
+	fi
+
+	sleep "$(cronjob_interval)"
```

Comment:
In the README you state that "-1" is a supported arg. What happens here if "-1" is supplied? Errors and a very fast infinite loop taking CPU? If we want to support disabling this job we should probably just exit 0 and change the `restart-condition` to `on-failure`.